### PR TITLE
Nit workflow fix to keep CI greener

### PR DESCRIPTION
**Pull request by cijothomas**
image

^ ->?

### CODE REVIEW

File: .github/workflows/dotnet-format.yml [hunk 1]
```diff
@@ -20,14 +14,13 @@ jobs:
...
-        dotnet-version: '7.0.x'
-        include-prerelease: true
+        dotnet-version: 6.0.x
 
     - name: Install format tool
       run: dotnet tool install -g dotnet-format
 
     - name: dotnet format
-      run: dotnet-format --folder --check
+      run: dotnet-format --folder --check
```
Comment (reyang) [inline .github/workflows/dotnet-format.yml:26]: add a new line?

**Review by reyang — APPROVED** [review]

**Comment by cijothomas** [comment]
image
success!

File: .github/workflows/dotnet-format-md.yml [hunk 2]
```diff
@@ -10,12 +10,13 @@ name: dotnet format
...
+    - '**.cs'
+    - '.editorconfig'
 
 jobs:
   check-format:
     runs-on: ubuntu-latest
 
     steps:
-      - run: 'echo "No build required"'
+      - run: 'echo "No build required"'
```
Comment (alanwest) [inline .github/workflows/dotnet-format-md.yml:22]: At first I thought to myself... 🐇 approved a PR without a newline at the end of the file?! BUT then I opened up the resolved comment https://github.com/open-telemetry/opentelemetry-dotnet/pull/3548#discussion_r938328292 😆. #3556

Comment (reyang) [reply .github/workflows/dotnet-format-md.yml:22]: Adding a cop here #3557.

**Pull request merged**